CMake/SlicerMacroBuildApplication.cmake

```diff
@@ -285,6 +286,8 @@ macro(slicerMacroBuildApplication)
     LICENSE_FILE
 
     TARGET_NAME_VAR
+
+    APPLICATION_DEFAULT_ARGUMENTS # space separated list
```

Comment:
Is this change intentional?

Reply:
Yes, this provide  way to specify additional default arguments. It is the variable `SLICERAPP_APPLICATION_DEFAULT_ARGUMENTS` concatenated to `_launcher_application_default_arguments`

It is space separated because is is handled as such in the launcher:
https://github.com/commontk/AppLauncher/blob/1367de4c6efde0c11e87835fb7245ea2b05074aa/Base/ctkAppLauncher.cpp#L272-L281